docs/quickstart.rst

```diff
@@ -53,10 +53,22 @@ Running the Vagrant box
 
 The image is available in https://atlas.hashicorp.com/atomicapp/boxes/dev . However you dont have to download the box image manually. The following steps will take care of it implicitly.
 
+*This Vagrant box is built for VirtualBox. *
+
+To run it on libvirt, a vagrant plugin, `vagrant-mutate <https://github.com/sciurus/vagrant-mutate>`_ must be installed.
+To install vagrant-mutate on *NIX system, run
+
+::
+   $vagrant plugin install vagrant-mutate
+
```

Comment:
I will suggest to add these as steps for Ubuntu as these are required for Ubuntu . Let me know what do u think?